assets/js/components/setup/SetupUsingProxyWithSignIn.js

```diff
@@ -384,9 +387,10 @@ export default function SetupUsingProxyWithSignIn() {
 														</p>
 													) }
 
-												{ ! analyticsModuleActive && (
-													<ActivateAnalyticsNotice />
-												) }
+												{ analyticsModuleAvailable &&
```

Comment:
Similarly, this check for `analyticsModuleAvailable` is required in conjunction with `! analyticsModuleActive` - it's not sufficient to simply rely on `! analyticsModuleActive`.

Again this is because we only want to handle the "not active" case when the module _does_ exist, in this case only showing the `ActivateAnalyticsNotice` when Analytics does exist to be activated

Reply:
Similarly, this check for `analyticsModuleAvailable` is required in conjunction with `! analyticsModuleActive` - it's not sufficient to simply rely on `! analyticsModuleActive`.

Again this is because we only want to handle the "not active" case when the module _does_ exist, in this case only showing the `ActivateAnalyticsNotice` when Analytics does exist to be activated

Reply:
Similarly, this check for `analyticsModuleAvailable` is required in conjunction with `! analyticsModuleActive` - it's not sufficient to simply rely on `! analyticsModuleActive`.

Again this is because we only want to handle the "not active" case when the module _does_ exist, in this case only showing the `ActivateAnalyticsNotice` when Analytics does exist to be activated

Reply:
Similarly, this check for `analyticsModuleAvailable` is required in conjunction with `! analyticsModuleActive` - it's not sufficient to simply rely on `! analyticsModuleActive`.

Again this is because we only want to handle the "not active" case when the module _does_ exist, in this case only showing the `ActivateAnalyticsNotice` when Analytics does exist to be activated